src/ctim/generators/id.cljc

```diff
@@ -17,18 +19,27 @@
 
 (def gen-host
   (gen/frequency
-   [[2 (chuck/string-from-regex
-        #"[a-zA-Z\d][-\da-zA-Z]{2,9}(\.[-\da-zA-Z]{3,10}){0,4}")]
-    [1 gen-ipv4-addr]]))
+   #?(:clj
+      [[2 (chuck/string-from-regex
+           #"[a-zA-Z\d][-\da-zA-Z]{2,9}(\.[-\da-zA-Z]{3,10}){0,4}")]
+       [1 gen-ipv4-addr]]
+
+      :cljs [[2 gen-ipv4-addr]
+             [1 gen-ipv4-addr]])))
 
 (def gen-port
   (gen/one-of
    [(gen/return nil)
     (gen/choose 1000 65535)]))
 
 (def gen-path
-  (chuck/string-from-regex
-   #"(\/([\w][-\w]{0,5}(\.[-\w]{1,4}){0,3})){1,3}"))
+  #?(:clj (chuck/string-from-regex
+           #"(\/([\w][-\w]{0,5}(\.[-\w]{1,4}){0,3})){1,3}")
+     :cljs
+     (gen/fmap (fn [[f s]]
+                 (str "/" f "/" s))
+               (gen/tuple (gen-str-3+ gen-char-alpha-lower)
+                          (gen-str-3+ gen-char-alpha-lower)))))
```

Comment:
I think you should just use the :cljs implementation for both here.  Maybe we can get rid of chuck.